gradle.properties

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright 2020 Square Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+org.gradle.jvmargs='-Dfile.encoding=UTF-8'
+android.useAndroidX=true
+
+# Required to publish to Nexus (see https://github.com/gradle/gradle/issues/11308)
+systemProp.org.gradle.internal.publish.checksums.insecure=true
+
+GROUP=com.squareup.radiography
+VERSION_NAME=1.7-SNAPSHOT
```

Comment:
This was 1.6-SNAPSHOT in the pom, but 1.6 was already released internally so it was wrong.